.coveragerc

```diff
@@ -337,6 +337,11 @@ omit =
     homeassistant/components/flick_electric/const.py
     homeassistant/components/flick_electric/sensor.py
     homeassistant/components/flock/notify.py
+    homeassistant/components/flukso/__init__.py
+    homeassistant/components/flukso/binary_sensor.py
+    homeassistant/components/flukso/const.py
+    homeassistant/components/flukso/discovery.py
+    homeassistant/components/flukso/sensor.py
```

Comment:
It is not compulsory, but since new integrations can take a while to get merged, I suggest that you take the opportunity to move to 100% coverage whilst the reviews are in progress.